src/_data/docs.yml

```diff
@@ -48,6 +48,8 @@
   items:
   - title: Tiltfile API Reference
     href: api.html
+  - title: Tiltfile API - os module
```

Comment:
we're probably going to want to do something different with modules in the sidebar, rather than throwing all these links in with other links -- maybe their own subsection, or another level of nesting? but this is fine for now.

Reply:
ya, totally agree. if we have enough of these, we're going to want to change the flow here.